include/PR/gbi.h

```diff
@@ -4699,7 +4699,7 @@ _DW({									\
 
 /* like gSPTextureRectangle but accepts negative position arguments */
 #define gSPScisTextureRectangle(pkt, xl, yl, xh, yh, tile, s, t, dsdx, dtdy) \
-{                                                                            \
+_DW({                                                                        \
```

Comment:
![despair](https://user-images.githubusercontent.com/7416381/235379935-8247c73b-22ef-4c94-a9ae-0604cc08441f.png)

Reply:
Fwiw this is the current only use in MM, and it doesn't get used at all in OoT.